debian/patches/0001-Compile-examples-with-lpthread.patch

```diff
@@ -0,0 +1,20 @@
+From: =?utf-8?q?Ond=C5=99ej_Sur=C3=BD?= <ondrej@sury.org>
+Date: Fri, 2 Dec 2016 14:02:43 +0100
+Subject: Compile examples with -lpthread
+
+---
+ examples/Makefile.am | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/examples/Makefile.am b/examples/Makefile.am
+index da993e4..93e4c5b 100644
+--- a/examples/Makefile.am
++++ b/examples/Makefile.am
+@@ -1,6 +1,6 @@
+ EXAMPLE_LDADD = libmongoc-1.0.la
+ if EXPLICIT_LIBS
+-EXAMPLE_LDADD += $(BSON_LIBS)
++EXAMPLE_LDADD += $(BSON_LIBS) -lpthread
```

Comment:
Is this a debian specific requirement?
I'd rather conditionally add it to the buildflags during configure then to maintain branches for something we should be doing anyway